tests/appsec/iast/source/test_get_request_uri.py

```diff
@@ -0,0 +1,41 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+
+from utils import context, coverage, bug
+from .._test_iast_fixtures import SourceFixture
+
+
+@coverage.basic
+class TestRequestURI:
+    """Verify that strings returned from getRequestURI API are tainted"""
+
+    source_fixture = SourceFixture(
+        http_method="GET",
+        endpoint="/iast/source/getrequesturi/test",
+        request_kwargs={},
+        source_type="http.request.uri",
+        source_name="/iast/source/getrequesturi/test",
```

Comment:
Wait. This is actually `http.request.path`...?

Reply:
I have added getPathInfo test and endpoint

Reply:
I have added getPathInfo test and endpoint